src/transformStateWithClones.js

```diff
@@ -1,13 +1,35 @@
 'use strict';
 
-/**
- * @param {Object} state
- * @param {Object[]} actions
- *
- * @return {Object[]}
- */
 function transformStateWithClones(state, actions) {
   // write code here
-}
+  const copy = { ...state };
+  const mainArray = [];
```

Comment:
The name of the variable should tell us about the information that is stored inside
```suggestion
  const copyState = { ...state };
  const states = [];
```